engine/gamesys/src/gamesys/components/comp_sprite.cpp

```diff
@@ -257,6 +235,12 @@ namespace dmGameSystem
         return result;
     }
 
+    static inline bool GetUseSlice9(const SpriteComponent* component)
+    {
+        return sum(component->m_Resource->m_DDF->m_Slice9) != 0 &&
+            component->m_Resource->m_DDF->m_SizeMode == dmGameSystemDDF::SpriteDesc::SIZE_MODE_MANUAL;
+    }
+
```

Comment:
Moved the check to a function, but maybe I should move the clarifying comment here as well 🤔

Reply:
We do this each frame, for each sprite.
We dereference the pointer 3 times (this evicting the caches)
I would like us to store this bit in the `component` to keep it closer to the rest of the data.